sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -857,19 +857,23 @@ object DDLUtils {
     }
   }
 
-  private[sql] def checkDataSchemaFieldNames(table: CatalogTable): Unit = {
+  private[sql] def checkDataColNames(table: CatalogTable): Unit = {
+    checkDataColNames(table, table.dataSchema.fieldNames)
+  }
+
+  private[sql] def checkDataColNames(table: CatalogTable, colNames: Seq[String]): Unit = {
```

Comment:
I think it's better to let a checking method return void. cc @dongjoon-hyun

Reply:
Do you mean `checkFieldNames` in [line 870](https://github.com/apache/spark/pull/19644/files#diff-54979ed5797b4a6193cf663dc23baca5R870) instead of `checkDataColNames` or `checkDataSchemaFieldNames`?

Reply:
+1!